src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
+
+      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
+        {comment.email}
+      </a>
+    </div>
+
+    <div className="CommentInfo__body">{comment.body}</div>
```

Comment:
```suggestion
export const CommentInfo = ({ comment }) => {
  const {name, ...} = comment;
  
  return (
  <div className="CommentInfo">
    <div className="CommentInfo__title">
      <strong className="CommentInfo__name">{name}</strong>

      {' by '}

      <a className="CommentInfo__email" href={`mailto:${email}`}>
        {email}
      </a>
    </div>

    <div className="CommentInfo__body">{body}</div>
```

use desctructuring here and in all same cases instead of overcwriting "comment" variable